### PR TITLE
Slider and Snackbar Added

### PR DESCRIPTION
**Pull request by sameep-s**
What's New:
Snack-bar component added. click [here](https://deploy-preview-10--aphro-css.netlify.app/components/snackbar.html) to see snack-bar
Basic slider added. click [here](https://deploy-preview-10--aphro-css.netlify.app/components/slider.html) to see slider.

### CODE REVIEW

**Review by Manju-sree — APPROVED** [review]
everything looks fine in code, you can merge

File: components/slider.html [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+      <div class="topbar-right-container">
+        <a href="https://github.com/sameep-s/Aphro_CSS-Library"
+          ><img
+            src="../assets/images/git-mark.png"
+            class="git-logo"
+            alt="github-logo"
+        /></a>
+      </div>
+    </div>
+    <div class="main-wrapper">
```
Comment (junaidshaikh-js) [inline components/slider.html]: for main content `main` element can be used

File: components/snackbar.html [hunk 2]
```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Documentation Page</title>
```
Comment (junaidshaikh-js) [inline components/snackbar.html]: title name should be meaningful

**Review by junaidshaikh-js — COMMENTED** [review]
Good Work!

File: components/slider.html (hunk 1 above)
Comment (ShreyashMahajan) [inline components/slider.html]: you can use more semantic here "main"

File: components/slider.html [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+      <!-- sidebar -->
+      <div class="sidebar-wrapper">
```
Comment (ShreyashMahajan) [inline components/slider.html]: sidebar ---> "aside"

**Review by ShreyashMahajan — APPROVED** [review]
looks good to merge

File: components/snackbar.html (hunk 2 above)
Comment (sameep-s) [reply components/snackbar.html]: i changed it.

File: components/slider.html (hunk 1 above)
Comment (sameep-s) [reply components/slider.html]: i changed it.

Comment (sameep-s) [reply components/slider.html]: i changed it.

**Pull request merged**